### PR TITLE
Idva6 496 userslist

### PR DESCRIPTION
**Pull request by kkonuganti-ch**
new feature to fetch associations for user in session

__JIRA Ticket Number__

### Type of change

* [ ] Bug fix
* [ ] New feature
* [ ] Breaking change
* [ ] Infrastructure change

### Pull request checklist

* [ ] I have added unit tests for new code that I have added
* [ ] I have added/updated functional tests where appropriate
* [ ] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)__

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/test/java/uk/gov/companieshouse/accounts/association/integration/AssociationsServiceTest.java
Comment (krishna-patel-ch) [inline src/test/java/uk/gov/companieshouse/accounts/association/integration/AssociationsServiceTest.java:1]: No new tests were added

File: src/test/java/uk/gov/companieshouse/accounts/association/controller/UserCompanyAssociationsTest.java
[the remark this answers is not in the document]
Comment (krishna-patel-ch) [reply src/test/java/uk/gov/companieshouse/accounts/association/controller/UserCompanyAssociationsTest.java:1]: The tests don't check the response thoroughly enough.

**Review by krishna-patel-ch — CHANGES_REQUESTED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/main/java/uk/gov/companieshouse/accounts/association/controller/UserCompanyAssociations.java
[the remark this answers is not in the document]
Comment (krishna-patel-ch) [reply src/main/java/uk/gov/companieshouse/accounts/association/controller/UserCompanyAssociations.java:1]: Line 57 change is not addressed

**Review by krishna-patel-ch — APPROVED** [review]

**Pull request merged**